frontends/web/src/utils/env.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+ /* eslint no-undef: "off" */
 export const debug = process.env.NODE_ENV === 'development';
```

Comment:
An alternative would be to set `node: true`, here https://github.com/digitalbitbox/bitbox-wallet-app/blob/4457dde3491a63ac184e00da9f0d328d9c2fe227/frontends/web/.eslintrc.js#L3

but node is only used during build, so I prefer this way.